datalad/core/distributed/clone.py

```diff
@@ -585,15 +585,125 @@ def postclonecfg_ria(ds, props):
                    if s.get('annex-externaltype', None) == 'ora'
     ]
     if not ria_remotes:
-        lgr.debug("Found no RIA special remote")
+        # nothing autoenabled. Let's check origin's config for
+        # datalad.ora-remote.uuid as stored by create-sibling-ria and enable try
+        # enabling that one.
+        lgr.debug("Found no autoenabled ORA special remote. Trying to look it"
+                  "up in source config ...")
```

Comment:
One of the lines needs a space.

Reply:
No idea why this isn't marked outdated. But this is done.